packages/validate/src/lib/validation.ts

```diff
@@ -617,6 +617,13 @@ function parseUrl(str: string) {
 	try {
 		return new URL(str)
 	} catch (error) {
+		if (str.startsWith('/') || str.startsWith('./')) {
+			try {
+				return new URL(str, 'http://example.com')
```

Comment:
Is including `https://example.com` correct here? Not sure I follow

Reply:
We first check if we have a fully qualified url (first try catch). If we don't we then check if we have a local image like in our `LocalImage` example: `/tldraw.png`. URL constructor only works with fully qualified urls, so we could either use the current base url or just a dummy one like `example.com` 🤷‍♂️

Reply:
Sounds good, let's land it!